packages/common/src/ui/layout/tables/components/Cells/TextInputCell/TextInputCell.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { CellProps } from '../../..';
+import { BasicTextInput } from '../../../../../components/inputs/TextInput';
+import { DomainObject } from '../../../../../../types';
+import { useDebounceCallback } from '../../../../../..';
```

Comment:
I think it doesn't matter - but given the issues with imports, I'd feel safer if these were referencing explicitly `columns` and `hooks`. helps a little with readability, knowing where things are sourced too

Reply:
Yes! good catch, thanks